CHANGELOG.md

```
@@ -67,9 +67,22 @@ Semantic Versioning where the repository publishes a release.
 ### Changed
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)